aws-datastore/src/test/java/com/amplifyframework/datastore/syncengine/SyncProcessorTest.java

```diff
@@ -100,6 +101,8 @@
     private SyncProcessor syncProcessor;
     private int errorHandlerCallCount;
     private int modelCount;
+    RetryHandler retryHandler;
```

Comment:
Should this be private?

```suggestion
    private RetryHandler retryHandler;
```